### PR TITLE
fix formatter for html and error in rest.run()

### LINKED ISSUE #164: Disable debug logs?
Is it possible to disable debug logs, some debug logs are not very helpful for user.
e.g.
```
[rest.nvim] [DEBUG 14:43:41] ...pack/packer/opt/rest.nvim/lua/rest-nvim/request/init.lua:179: Searching pattern starting from 23
```

### PR DESCRIPTION
**Pull request by udayvir-singh**
Fixes error caused by html formatter running even when tidy is not available on the system. Also fixes docs related to disabling formatters.

fixes #164

### CODE REVIEW

**Comment by udayvir-singh** [comment]
Now this PR also fixes #164 and fixes nasty error on rest.run() due to incorrect handling of opts.

File: lua/rest-nvim/init.lua [hunk 1]
```diff
@@ -21,7 +21,7 @@ rest.run = function(verbose)
     return
   end
 
-  return rest.run_request(result, verbose)
+  return rest.run_request(result, {verbose = verbose})
```
Comment (ray-x) [inline lua/rest-nvim/init.lua:24]: this will fix a breaking change.

**Review by ray-x — COMMENTED** [review]
Thanks. the changes fix https://github.com/rest-nvim/rest.nvim/issues/164

**Review by ray-x — APPROVED** [review]
LGTM

**Comment by teto** [comment]
you can drop the second commit. I think it's best to throw an error if tidy is not available, so nothing to change on that side, what I am curious about is why `html = nil` didn't work (that's the way to remove entries from lua arrays). I believe it might be due to ` config = vim.
tbl_deep_extend("force", config, user_configs)`

**Pull request closed without merging**